timelock-corruption-detection/src/main/java/com/palantir/history/PaxosLogHistoryProvider.java

```diff
@@ -61,16 +80,131 @@ private Long getOrInsertVerificationState(NamespaceAndUseCase namespaceAndUseCas
                 namespaceAndUseCase.namespace(), namespaceAndUseCase.useCase());
     }
 
-    public List<CompletePaxosHistoryForNamespaceAndUseCase> getHistory() {
+    /**
+     * Gets history from all the nodes for all unique namespace, useCase tuples since last verified sequence numbers
```

Comment:
nit: `(namespace, useCase) tuples` (with parentheses)

It's also useful to explain what `last verified sequence numbers` are here (specifically, the highest sequence number that was verified since the last time the bounds were reset?)